### PR TITLE
Event api endpoint

### PR DESCRIPTION
**Pull request by Tiseno**
Needs some stuff in companies branch for migrations to work with images and stuff.

### CODE REVIEW

File: events/models.py [hunk 1]
```diff
@@ -20,17 +24,26 @@ class Event(models.Model):
...
     submission_mail_subject = models.TextField(blank=True)
     submission_mail_body = models.TextField(blank=True)
+    image = models.ImageField(
+            upload_to=random_path('events', 'image'), blank=True)
 
     def __str__(self):
         return '%s'%(self.name)
 
+    def save(self, *args, **kwargs):
+        super(Event, self).save(*args, **kwargs)
```
Comment (mictab) [inline events/models.py]: Why is save called twice? :)

File: lib/image.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 from django.core.files.uploadedfile import SimpleUploadedFile
 import os
 
+def path():
```
Comment (mictab) [inline lib/image.py]: Why is this here? :)

File: events/models.py (hunk 1 above)
Comment (Tiseno) [inline events/models.py]: Because the image is first saved and then converted and resized. This is a temporary solution and we are looking for a solution to it but for now it is OK.

Comment (mictab) [inline events/models.py]: Cool.

**Comment by mictab** [comment]
@walaro @purmonen Can you take a final look and merge? :)
